packages/components/src/components/tabs/_tabs.scss

```diff
@@ -18,370 +18,790 @@
 /// @access private
 /// @group tabs
 @mixin tabs {
-  .#{$prefix}--tabs {
-    @include reset;
-    @include type-style('body-short-01');
-
-    display: flex;
-    width: 100%;
-    height: auto;
-    min-height: rem(40px);
-    color: $text-01;
-  }
+  // TODO: remove block in next major release
+  .#{$prefix}--tabs--stable {
```

Comment:
This would be a major breaking change for us (Angular). This would result in none of the tabs styles applying to our current tabs implementation. Given that you're already scoping the scrollable tabs (`bx--tabs-scrollable`) I believe it should be sufficient to leave `bx--tabs` un-scoped and just override classes as needed within the `bx--tabs-scrollable` scope.

Reply:
good catch, I think this was left over from the other PR which will probably be closed in favor of this one

Reply:
@cal-smith thanks again for taking a look, it's great to have your feedback with regression and compatibility testing